.github/workflows/lints.yml

```diff
@@ -30,6 +30,8 @@ jobs:
     runs-on: ubuntu-20.04
     if: |
       !contains(github.event.pull_request.labels.*.name, 'ci-condition: skip')
+    container:
+      image: timberio/ci_image:latest
```

Comment:
So it was my general plan to build the environment via `make environment` in a pre-step of the CI (when needed) and then have all further steps use this feature to use that environment. :)

Reply:
Per workflow or per job?

Reply:
Per workflow.

Since the long term goal is to have our release and test process be similar, this would suggest having all our testing pipeline on one workflow and having the `make environment` step create the initial cached image for later workflow tasks.

Reply:
Yeah - I considered that approach and wondering if a scheduled build of the image (nightly?) was a better async approach so we're not building the image for each run.

Reply:
I worry that isn't effective, since our dependencies might change between PRs and render the builds for the day failing.

(For example, we introduce a dependency on `cmake` or `openssl` or `sasl`, which has happened before)